src/components/Contentful/DynamicHero.vue

```diff
@@ -173,6 +173,24 @@ export default {
 				link: this.heroText?.primaryCtaLink ?? '',
 			};
 		},
+		heroBackground() {
+			return this.content?.contents?.find(({ key }) => key.indexOf('background') > -1);
+		},
+		heroBackgroundStyle() {
+			// Creates an object with conditional keys if heroBackground contains those elements
+			const styleObject = {
+				...(this.heroBackground?.backgroundColor && {
+					'background-color': this.heroBackground?.backgroundColor
+				}),
+				...(this.heroBackground?.backgroundImage?.file?.url && {
+					'background-image': `url("${this.heroBackground?.backgroundImage?.file?.url}")`
```

Comment:
The caveat here is that this is getting the full size image from contentful. Not sure if theres a way to optimize this.

Reply:
I think this is a good first version. Ultimately, this may be a use case for our KvContentfulImage or event a responsive image set...